firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -25,8 +31,8 @@
 import com.google.android.gms.tasks.Tasks;
 import com.google.firebase.FirebaseApp;
 import com.google.firebase.FirebaseOptions;
-import com.google.firebase.installations.local.FiidCache;
-import com.google.firebase.installations.local.FiidCacheEntryValue;
+import com.google.firebase.installations.local.PersistedFid;
+import com.google.firebase.installations.local.PersistedFidEntryValue;
```

Comment:
Probably driving you crazy here, so please feel free to ignore this comment:
How about "PersistedFidEntry" instead of "PersistedFidEntryValue"?

Reply:
Done. Thanks.